diff -urN libpst-0.6.45/m4/ax_boost_python.m4 libpst-0.6.45.new//m4/ax_boost_python.m4
--- libpst-0.6.45/m4/ax_boost_python.m4	2009-12-11 17:46:02.000000000 +0100
+++ libpst-0.6.45.new//m4/ax_boost_python.m4	2010-02-25 18:30:56.813698020 +0100
@@ -63,7 +63,7 @@
  AC_LANG_CPLUSPLUS
  CPPFLAGS_SAVE=$CPPFLAGS
  if test x$PYTHON_INCLUDE_DIR != x; then
-   CPPFLAGS=-I$PYTHON_INCLUDE_DIR $CPPFLAGS
+   CPPFLAGS="-I$PYTHON_INCLUDE_DIR $CPPFLAGS"
  fi
  AC_COMPILE_IFELSE(AC_LANG_PROGRAM([[
  #include <boost/python/module.hpp>
