--- libpst-0.5.1/Makefile~	2004-11-17 18:16:02.000000000 +0200
+++ libpst-0.5.1/Makefile	2006-01-04 14:48:21.000000000 +0200
@@ -6,10 +6,10 @@
 
 #---------------- Do not modify below this point ------------------
 
-INSTALL_DIR     := $(INSTALL) -p -d -o root -g root -m 0755
-INSTALL_FILE    := $(INSTALL) -p    -o root -g root -m 0644
-INSTALL_PROGRAM := $(INSTALL) -p    -o root -g root -m 0755 # -s
-INSTALL_SCRIPT  := $(INSTALL) -p    -o root -g root -m 0755
+INSTALL_DIR     := $(INSTALL) -p -d
+INSTALL_FILE    := $(INSTALL) -p
+INSTALL_PROGRAM := $(INSTALL) -p
+INSTALL_SCRIPT  := $(INSTALL) -p
 
 VERSION = $(shell cat VERSION)
 
--- libpst-0.5.1/Makefile~	2006-01-04 14:49:05.000000000 +0200
+++ libpst-0.5.1/Makefile	2006-01-04 14:50:09.000000000 +0200
@@ -59,7 +59,6 @@
 	$(INSTALL_DIR) $(DESTDIR)$(PREFIX)/share/man/man1
 	$(INSTALL_FILE) readpst{,log}.1 $(DESTDIR)$(PREFIX)/share/man/man1/
 	$(INSTALL_DIR) $(DESTDIR)$(PREFIX)/share/doc/libpst
-	$(INSTALL_FILE) $(DOCS) $(DESTDIR)$(PREFIX)/share/doc/libpst/
 
 uninstall:
 	-rm -f $(DESTDIR)$(PREFIX)/bin/readpst{,log}
